### ISSUE TITLE
Docker build rule requires that debs end in .deb and doesn't support .udeb.

### ISSUE THREAD

**Issue by endobson** [bazelbuild/bazel#612]
I'm trying to use a udeb, in particular libc (https://packages.debian.org/sid/libc6-udeb). I am needing to use a genrule to move the filename over to a .deb file otherwise I get an error. But since the contents are the same the check on the filename seems overly restrictive.

BUILD file
```
docker_build(
  name = "minimal_deb",
  debs = [":libc.deb"]
)

genrule(
  name = "libc-deb",
  outs = ["libc.deb"],
  srcs = ["@libc-udeb//file"],
  cmd = "cp $(location @libc-udeb//file) $@"
)
```

WORKSPACE
```
http_file(
  name = "libc-udeb",
  url = "http://http.us.debian.org/debian/pool/main/g/glibc/libc6-udeb_2.19-22_amd64.udeb",
  sha256 = "a26c32f02100929dd9fda0c19947b293ba228ab09d0e7fcad6b0bc28e42ca01f"
)
```

Error message
```
endobson@yggdrasil () ~/tmp/bazel-project % bazel build --verbose_failures :minimal_deb                                                                                                                                                (0)
ERROR: /Users/endobson/tmp/bazel-project/BUILD:17:1: in debs attribute of docker_build_ rule //:minimal_deb: '@libc-udeb//file:file' does not produce any docker_build_ debs files (expected .deb). Since this rule was created by the macro 'docker_build', the error might have been caused by the macro implementation in /Users/endobson/tmp/bazel-project/BUILD:17:1.
```

**Comment by damienmg** [comment]
This is certainly not the latest errors, can you past the whole output?

**Comment by endobson** [comment]
I'm not sure what you want from the whole output, I just trimmed the boring lines after the interesting ones.

```
endobson@yggdrasil () ~/tmp/bazel-project % bazel build --verbose_failures :minimal_deb                                                                                                                                                (1)
ERROR: /Users/endobson/tmp/bazel-project/BUILD:12:1: in debs attribute of docker_build_ rule //:minimal_deb: '@libc-udeb//file:file' does not produce any docker_build_ debs files (expected .deb). Since this rule was created by the macro 'docker_build', the error might have been caused by the macro implementation in /Users/endobson/tmp/bazel-project/BUILD:12:1.
ERROR: Analysis of target '//:minimal_deb' failed; build aborted.
INFO: Elapsed time: 0.093s
```

My repository that is having the issue
https://github.com/endobson/minimal-racket/commit/9133bcfe0fbe5be468f70d2cf3e6fb94779633ae

```
endobson@yggdrasil () ~/tmp/bazel-project % bazel version                                                                                                                                                                              (2)
Build label: 0.1.1
Build target: bazel-out/local_darwin-fastbuild/bin/src/main/java/bazel-main_deploy.jar
Build time: Thu Oct 15 20:51:11 2015 (1444942271)
Build timestamp: 1444942271
Build timestamp as int: 1444942271
```

This matches the release that is here: https://github.com/bazelbuild/bazel/releases, are you saying that HEAD has this fixed?

**Comment by damienmg** [comment]
Oh I understood, easy to fix :)

**Comment by damienmg** [comment]
This bug was fixed by b84f540eff8b45acaae15c0cd11d0342b32917c4

I have messed up with the bug number, sorry for that.
